### ISSUE TITLE
Accessibility issues with docs

### ISSUE THREAD

**Issue by coliff** [trimble-oss/website-modus.trimble.com#144]
Spotted a few minor issues with the docs:

- Pagination examples are missing aria-label for the icons
- visual bug on Accessibility page list (see: https://modus.trimble.com/foundations/accessibility/#usability)
- 'skip to main' link doesn't have a `main` link to skip to on Components page
- There are duplicate `aria-label="Search"` forms. We should remove label from both and set it once with JS
- the `<h5>` heading on right-side causes accessibility issues
